### PR TITLE
Added hangman game

### PR DESCRIPTION
**Pull request by pranjay-poddar**
linked issue #17

### CODE REVIEW

**Review by kanishkaa24 — APPROVED** [review]
Looks good to me!

File: index.html [hunk 1]
```diff
@@ -47,46 +47,45 @@ <h3 id="pop">Have a break? Play a game!</h3>
...
             </div>
             <div class="col-12 col-md-3">
                 <a href="wallbreaker.html">
-                 <img src="static/images/WallBreaker/WallBreaker.png" alt="Wall Breaker" width="200" height="150">
+                    <img src="static/images/WallBreaker/WallBreaker.png" alt="Wall Breaker" width="200" height="150">
                 </a>
                 <p class="mt-3"><strong>Wall Breaker</strong></p>
             </div>
-        </div>
-        <div class="row game justify-content-left">
```
Comment (geekymeeky) [inline index.html:72]: I think these lines have been mistakenly deleted. Kindly correct it.

File: index.html [hunk 2]
```diff
@@ -47,46 +47,45 @@ <h3 id="pop">Have a break? Play a game!</h3>
...
             <div class="col-12 col-md-3">
-                <a href="snakegame.html"><img width="200" height="150"
-                    src="https://i.pinimg.com/originals/b0/20/57/b020573677505d9d5dde2fdf21e515aa.png" alt="image"></a>
-            <p class="mt-3"><strong>Snake Game</strong></p>
+                <a href="./hangman.html"><img width="200" height="150" src="./static/images/hangman/hangman.png" alt="image"></a>
+                <p class="mt-3"><strong>Hangman Game</strong></p>
             </div>
+
         </div>
-    </div>
```
Comment (geekymeeky) [inline index.html:85]: Some of the div tags are not closed properly. Thus causing layout issues. Kindly fix it. image

**Review by geekymeeky — CHANGES_REQUESTED** [review]
You can push the required changes by committing to your PR branch.

**Comment by pranjay-poddar** [comment]
Did all the required changes please check.

**Review by geekymeeky — CHANGES_REQUESTED** [review]
@pranjay-poddar It isn't working. Kindly run the changes before pushing them.
image

**Comment by pranjay-poddar** [comment]
image


its working perfectly in my local machine pls check, already checked it many times and found no errors in it.

**Comment by geekymeeky** [comment]
Fix #17

**Pull request merged**